boltdesignsystem/bolt
# [DO NOT MERGE] Remove unused '@bolt/build-tools' dependency from micro-journeys

## Jira

n/a

## Summary

Remove unused `@bolt/build-tools` dependency from Microjourneys, causing warning when building Bolt.

## Details

If a package includes a dependency with `bolt` in the name and that dependency is not included in that environment's `.boltrc` file (or as a dependency of a package in that file), a warning is logged:
```
Warning! Some of the components being compiled have Bolt dependencies and/or peerDependencies that appear to be missing from your build configuration.

To prep for the upcoming Bolt v3.0 release, you'll need to add these packages to the components: { global: [] } section in your /Users/morsd/sites/bolt-backup/docs-site/.boltrc.js file to ensure that Twig namespaces are properly registered, any related Sass and JS code is compiled and loaded, and to have the fastest front-end web performance.

[ '@bolt/build-tools' ]
```

Removing `@bolt/build-tools` from `@bolt/micro-journeys` fixes the warning. `@bolt/build-tools` does not appear to be used anywhere in `@bolt/micro-journeys` so this seems to be a win-win.

Even if `@bolt/build-tools` is required downstream it should be required separately. @remydenton can you think of any reason not to remove this?

## How to test

- Run locally and verify there is no warning on `yarn start`
- Verify micro-journeys works as expected

## Release notes
Unused `@bolt/build-tools` dependency removed from `@bolt/micro-journeys`.
